com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -181,7 +181,16 @@ internal void ChangeOwnership(NetworkObject networkObject, ulong clientId)
             }
         }
 
-        // Only ran on Client
+        /// <summary>
+        /// Only run on the client 
+        /// </summary>
+        /// <param name="softCreate"></param>
+        /// <param name="prefabHash"></param>
+        /// <param name="ownerClientId"></param>
+        /// <param name="parentNetworkId"></param>
+        /// <param name="position"></param>
+        /// <param name="rotation"></param>
+        /// <returns></returns>
```

Comment:
I'd suggest removing these empty blocks if we don't specifically xmldoc them (but still keep the main `<summary>` block only).
```suggestion
```